openadapt/record.py

```diff
@@ -467,6 +493,35 @@ def performance_stats_writer (
     logger.info("performance stats writer done")
 
 
+def memory_writer(
+    recording_timestamp: float, terminate_event: multiprocessing.Event, record_pid: int
+):
+    utils.configure_logging(logger, LOG_LEVEL)
+    utils.set_start_time(recording_timestamp)
+    logger.info("Memory writer starting")
+    signal.signal(signal.SIGINT, signal.SIG_IGN)
+    process = psutil.Process(record_pid)
+
+    while not terminate_event.is_set():
+        mem_usage = getattr(process, "memory_info")()[0] / BYTE_TO_MB
```

Comment:
@angelala3252 can you please clarify why it is necessary to use `getattr` here instead of just `process.memory_info`?